frameworks/Python/API-Hour/hello/hello/servers/yocto_http.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+import ujson
+
+from ..utils import generate_http_response
+
+class YoctoHttpJson(asyncio.Protocol):
+    def connection_made(self, transport):
+        self.transport = transport
+
+    def data_received(self, data):
+        # self.transport.write(data)
+        payload = ujson.dumps({'message': 'Hello, World!'})
+        self.transport.write(generate_http_response(payload))
+
+class YoctoHttpText(asyncio.Protocol):
+    def connection_made(self, transport):
+        self.transport = transport
+
+    def data_received(self, data):
+        # self.transport.write(data)
+        payload = 'Hello, World!'
+        self.transport.write(generate_http_response(payload, 'text/plain; charset=UTF-8'))
```

Comment:
> plaintext test uses pipelined request.
> Skipping HTTP may not work.

You're certainly right, I'll dig that. I need to find the `wrk` command line used for this test.

> I feel skipping HTTP is less than "stripped". "stripped" means skip some default middleware in framework.

I'm agree with you if I've categorized this test as micro or full stack framework. But for theses tests, I've flagged that as `Platform`.
At least for me, it's the stripped version for a plateform.
The idea behind theses tests is to see the maximum it should be possible to have with AsyncIO in pure Python, like you have with wsgi, not to be used on production. "Platform" and "Stripped" are here as warnings.